--- l10n_ar_wsafip_fe/invoice.py
+++ l10n_ar_wsafip_fe/invoice.py
@@ -192,8 +192,8 @@
                         'Id': tax.tax_code_id.parent_afip_code,
                         'Desc': tax.tax_code_id.name,
                         'BaseImp': tax.base_amount,
-                        'Alic': (tax.tax_amount / tax.base_amount),
-                        'Importe': tax.tax_amount,
+                        'Alic': round((tax.tax_amount / tax.base_amount), 2),
+                        'Importe': tax.tax_amount
                     })
             else:
                 raise Warning(_('TAX without tax-code!\
