--- l10n_ar_bank/wizard/cache.py
+++ l10n_ar_bank/wizard/cache.py
@@ -5,7 +5,7 @@
 from geopy import geocoders
 from types import GeneratorType
 
-gc = geocoders.Google('ABQIAAAAMWm7ddpoRV3HO0u7NtA_IhRTfPMBNX3pvExQyYBKj7aZZJK5lxQYw0LDgWXedvepzKpGxQKf-kmN3A')
+gc = geocoders.GoogleV3('ABQIAAAAMWm7ddpoRV3HO0u7NtA_IhRTfPMBNX3pvExQyYBKj7aZZJK5lxQYw0LDgWXedvepzKpGxQKf-kmN3A')
 
 def urlopen(url):
     print >> sys.stderr, "Urlopen reading:", url
