--- a/models/orders.py	2020-06-24 16:18:30.238650175 -0300
+++ b/models/orders.py	2020-06-24 18:30:19.155767000 -0300
@@ -815,7 +815,7 @@
         #
         meli = Meli(client_id=CLIENT_ID,client_secret=CLIENT_SECRET, access_token=ACCESS_TOKEN, refresh_token=REFRESH_TOKEN )
 
-        orders_query = "/orders/search?seller="+company.mercadolibre_seller_id+"&sort=date_desc"
+        orders_query = "/orders/search?seller="+str(company.mercadolibre_seller_id)+"&sort=date_desc"
 
         if (offset):
             orders_query = orders_query + "&offset="+str(offset).strip()
