--- a/DocumentConverter.py
+++ b/DocumentConverter.py
@@ -38,10 +38,6 @@
 from com.sun.star.beans import UnknownPropertyException
 from com.sun.star.lang import IllegalArgumentException, DisposedException
 from com.sun.star.io import XOutputStream
-from com.sun.star.document.UpdateDocMode import QUIET_UPDATE
-from com.sun.star.document.MacroExecMode import NEVER_EXECUTE
-from com.sun.star.style.BreakType import PAGE_AFTER, PAGE_BEFORE, PAGE_BOTH
-from com.sun.star.text.ControlCharacter import PARAGRAPH_BREAK, APPEND_PARAGRAPH
 
 SECTIONMAXLEVEL = 10 # Just to make sure we do not go into endless loop
 

