--- a/padron.py	2020-06-29 13:30:30.000000000 -0300
+++ b/padron.py	2020-07-13 23:54:12.956618000 -0300
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf8 -*-
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by the
@@ -34,7 +34,7 @@
 from email.utils import formatdate
 import sys
 import warnings
-from .utils import leer, escribir, N, A, I, get_install_dir, safe_console, \
+from pyafipws.utils import leer, escribir, N, A, I, get_install_dir, safe_console, \
     inicializar_y_capturar_excepciones_simple, WebClient, norm, \
     exception_info
 
