--- pyafipws/utils.py
+++ pyafipws/utils.py
@@ -52,10 +52,9 @@
     import httplib2
     # corregir temas de negociacion de SSL en algunas versiones de ubuntu:
     import platform
-    dist, ver, nick = platform.linux_distribution() if sys.version > (2, 6) else ("", "", "")
+    dist, ver, nick = platform.linux_distribution()
     from pysimplesoap.client import SoapClient
-    monkey_patch = httplib2._ssl_wrap_socket.__module__ != "httplib2"
-    if dist == 'Ubuntu' and ver == '14.04' and not monkey_patch:
+    if dist == 'Gentoo Base System':
         import ssl
         def _ssl_wrap_socket(sock, key_file, cert_file,
                              disable_validation, ca_certs):
