--- l10n_ar_wsafip_fe/wsafip_server.py
+++ l10n_ar_wsafip_fe/wsafip_server.py
@@ -392,7 +392,7 @@
 
             for tc in tax_list:
                 tax_code_ids = tax_code_obj.search(
-                    cr, uid, [('name', '=like', '% ' + tc['name'])])
+                    cr, uid, [('name', '=like', '%' + tc['name'])])
                 _logger.debug(
                     "Tax '%s' match with %s" % (tc['name'], tax_code_ids))
                 if tax_code_ids:

